src/screens/ProfileScreen/elements/BottomSheetBio.js

```diff
@@ -52,28 +52,42 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[400],
     fontWeight: 'bold',
     fontSize: 24,
-    color: colors.black,
+    color: COLORS.black,
     marginBottom: dimen.normalizeDimen(12)
   },
   description: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
-    color: colors.gray,
+    color: COLORS.blackgrey,
     marginTop: 7
   },
   errorText: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
-    color: colors.red,
-    marginTop: dimen.normalizeDimen(7)
+    color: COLORS.redalert,
+    marginTop: 7
   },
   button: {
-    backgroundColor: colors.bondi_blue
+    backgroundColor: COLORS.bondi_blue,
+    marginTop: dimen.normalizeDimen(7)
   },
   textStyling: {
     fontFamily: fonts.inter[600],
     fontSize: 18,
-    color: colors.white
+    color: COLORS.white
+  },
+  input: {
+    backgroundColor: COLORS.lightgrey,
+    paddingVertical: 16,
+    paddingHorizontal: 12,
+    height: 150,
+    justifyContent: 'flex-start',
+    overflow: 'scroll',
+    borderRadius: 8,
+    fontFamily: fonts.inter[500],
+    fontSize: 14,
+    color: COLORS.black,
+    lineHeight: 24
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are primarily related to the color scheme of the application. The old `colors` module has been replaced with a new `COLORS` module from the `theme` directory. This change is consistent and does not seem to introduce any issues. However, it's important to ensure that the new `COLORS` module contains all the color constants used in this file. If any color constant is missing, it could potentially cause a runtime error.
```diff
-    color: colors.black,
+    color: COLORS.black,

-    color: colors.gray,
+    color: COLORS.blackgrey,

-    color: colors.red,
+    color: COLORS.redalert,

-    backgroundColor: colors.bondi_blue
+    backgroundColor: COLORS.bondi_blue,

-    color: colors.white
+    color: COLORS.white,

-    backgroundColor: colors.lightgrey,
+    backgroundColor: COLORS.lightgrey,

-    color: colors.black,
+    color: COLORS.black,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->